_usage/query_planning.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: default
+title: Reviewing a Query Plan
+nav_order: 8
+---
+
+## Reviewing a Query Plan
+
+Babelfish version 2.1.0 includes support for SET statements that allow you to review estimated and actual (executed) query plans. This allows you to use the TDS port to identify and refine slow-performing queries. The functions can display an query plan for:
+
+- `SELECT`, `INSERT`, `UPDATE`, and `DELETE` statements
+- Nested procedures, multiple statements in a batch
+- Variable declarations (for example, `DECLARE @t TABLE (a int, b int);`)
+
+The SET statements *do not* support query plan retrieval from functions, control flows, and cursors.
+
+You can use a SET statement on the TDS port to turn on/off the following functions: 
+
+- SET BABELFISH_STATISTICS PROFILE {ON|OFF} to display the query plan used to execute a statement. The command implements the behavior of the [PostgreSQL EXPLAIN ANALYZE statement] (https://www.postgresql.org/docs/current/using-explain.html#USING-EXPLAIN-ANALYZE).
+    
+- SET BABELFISH_SHOWPLAN_ALL {ON|OFF} to display estimated execution plans for a statement without performing the command. The command implements the behavior of the [PostgreSQL EXPLAIN statement](https://www.postgresql.org/docs/current/using-explain.html#USING-EXPLAIN-BASICS).
+
+Please note that you should not enable both SET statements at the same time.
+
+The following example turns on query planning, displays the query results and profile, and then turns off query planning:
```

Comment:
it does not turn on/off query planning (= the query optimizer). It turns on/off displaying of the query plan.

Reply:
How about:  In the following example, SET BABELFISH_STATISTICS PROFILE ON instructs the server to display the query results and plan for the SELECT statement that follows. SET BABELFISH_STATISTICS PROFILE OFF then stops the server from displaying the results and plans from any subsequent queries: